snippets/facets.liquid

```diff
@@ -41,7 +41,7 @@
             {%- for filter in results.filters -%}
               {%- assign total_active_values = total_active_values | plus: filter.active_values.size -%}
               {% case filter.type %}
-              {% when 'list' %}
+              {% when 'boolean' or 'list' %}
```

Comment:
Looks like using “or” is the preferred syntax here: https://github.com/Shopify/liquid/blob/master/lib/liquid/locales/en.yml#L8

Reply:
Interesting. How'd you come across that? Memory recall? I didn't see the ["theme linter"](https://github.com/Shopify/dawn/runs/5451041955?check_suite_focus=true) catch it.